site/releases/2024-feb-14/highlights.qmd

```diff
@@ -0,0 +1,236 @@
+---
+title: "February 14, 2024"
+---
+
+## Release highlights
+
+### ValidMind Developer Framework (v1.26.6)
+
+<!---[SC-2706] Documentation templates should allow rendering more than one unique metric result for the same metric by @AnilSorathiya in [#322](https://github.com/validmind/developer-framework/pull/322)--->
+#### Documentation templates now support multiple unique results for the same metric
```

Comment:
```suggestion
#### Documentation templates now tracking each metric result with a unique identifier
```

Reply:
^ The suggestion is missing a connecting verb. "now tracking" should be "now supports tracking"?

Reply:
Changed to `Documentation templates now supports tracking each metric result with a unique identifier`, as suggested by @cachafla.